include/alpaka/acc/AccOacc.hpp

```diff
@@ -62,40 +60,32 @@ namespace alpaka
 #    endif
 
     //! The OpenACC accelerator.
-    template<
-        typename TDim,
-        typename TIdx>
-    class AccOacc final :
-        public bt::IdxBtLinear<TDim, TIdx>,
-        public AtomicHierarchy<
-            AtomicOaccBuiltIn,    // grid atomics
-            AtomicOaccBuiltIn,    // block atomics
-            AtomicOaccBuiltIn     // thread atomics
-        >,
-        public math::MathStdLib,
-        public rand::RandStdLib,
-        public TimeStdLib,
-        public warp::WarpSingleThread,
-        // NVHPC calls a builtin in the STL implementation, which fails in OpenACC offload, using fallback
-        public IntrinsicFallback,
-        public concepts::Implements<ConceptAcc, AccOacc<TDim, TIdx>>,
-        public concepts::Implements<ConceptWorkDiv, AccOacc<TDim, TIdx>>,
-        public concepts::Implements<ConceptBlockSharedDyn, AccOacc<TDim, TIdx>>,
-        public concepts::Implements<ConceptBlockSharedSt, AccOacc<TDim, TIdx>>,
-        public concepts::Implements<ConceptBlockSync, AccOacc<TDim, TIdx>>,
-        public concepts::Implements<ConceptIdxGb, AccOacc<TDim, TIdx>>
+    template<typename TDim, typename TIdx>
+    class AccOacc final
+        : public bt::IdxBtLinear<TDim, TIdx>
+        , public math::MathStdLib
+        , public rand::RandStdLib
+        , public TimeStdLib
+        , public warp::WarpSingleThread
+        ,
```

Comment:
Is this a result of clang-format doing weird things?

Reply:
@jkelling Ping.

Reply:
After adding the concepts at the end and removing the `AtomicHierarchy`, I just rand git-clang-format of clang 11 on it. The CI clang-format appears to be happy with it, too.

I do not recall if changed anything else that could have triggered clang-format to change the comma position, although I prefer this style because adding and removing base-classes at the end by changing only one line.

If you are asking about the lone comma specifically, this is because the comment was before the base it belongs to in the old version, clang-format cannot change this.